From 72dcf46d67582331ee344ac88e79c55702c0558a Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 9 Dec 2021 18:48:59 +0100
Subject: [PATCH] Revert "Revert "turris-timezone: temporary fix for l18n""

It seems it is necessary again. We need to fix this in upstream
properly.
---
 utils/turris-timezone/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/utils/turris-timezone/Makefile b/utils/turris-timezone/Makefile
index a805547f4..d36059660 100644
--- a/utils/turris-timezone/Makefile
+++ b/utils/turris-timezone/Makefile
@@ -15,7 +15,6 @@ PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/foris-controller/$(PKG_NAME)/-/archive/v$(PKG_VERSION)/
 PKG_HASH:=0042a108d81b56c414e229638d432c15acc75ed1591a68ab213ad21db657441c
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-v$(PKG_VERSION)
-HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=l18n
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
 PKG_LICENSE:=GPL-3.0-or-later
-- 
2.37.1

